examples/illustrations/device/device.json

```diff
@@ -4,19 +4,20 @@
         "kb": "http://example.org/kb/",
         "acme": "http://custompb.acme.org/core#",
         "draft": "http://example.org/draft#",
-        "olo": "http://purl.org/ontology/olo/core#",
         "uco-core": "https://unifiedcyberontology.org/ontology/uco/core#",
+        "uco-location": "https://unifiedcyberontology.org/ontology/uco/location#", 
         "uco-observable": "https://unifiedcyberontology.org/ontology/uco/observable#",
         "xsd": "http://www.w3.org/2001/XMLSchema#"
     },
     "@graph": [
         {
-            "@id": "kb:forensic_lab_computer1",
+            "@id": "kb:forensic_lab_computer1-uuid",
             "@type": "uco-observable:ObservableObject",
-            "location": "kb:forensic_lab1",
+            "uco-location:Location": [{"kb:forensic_lab1-uuid"}],
```

Comment:
Two things about the left-hand-side of the colon:
1. `uco-location:Location` is a class, not a property.  Relatedly, the only `*:location` property is `action:location`, which is not appropriate on this line.
2. This was intentionally left as an unnamespaced concept, because in this example file we need to assign the device's location with a `Relationship` object.  It's not currently clear what `Relationship`, or vocabulary element, should be used to do that; we just had a lengthy discussion about this in this morning's OC meeting on Locations.
So, please revert the change to the left-hand-side.

On the right-hand-side, this is not legal JSON: JSON doesn't have Python-ic sets.  It should be a singleton object reference, spelled `{"@id": "kb:forensic_lab1-uuid"}`, noting the `@id` key.
If there were more than one thing to list for the `location` property, it would be appropriate to use a JSON array.  Since we just have the one object, let the right-hand-side be a single JSON dictionary.
Also, please 4-space indent the dictionary as done elsewhere in the file.  (`python3 -m json.tool` is the current whitespace normalizer.)

Reply:
I'll make changes to this now thanks for review